documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2239,7 +2239,29 @@ To capture large object values and serialized them in change events, set this op
 |[[oracle-property-rac-nodes]]<<oracle-property-rac-nodes, `+rac.nodes+`>>
 |No default
 |A comma-separated list of Oracle Real Application Clusters (RAC) node host names or addresses.
-This field is required to enable use with Oracle RAC.
+This field is required to enable Oracle RAC support.
+Specify the list of RAC nodes by using one of the following methods:
+
+* Specify a value for xref:oracle-property-database-port[`database.port`], and use the specified port value for each address in the `rac.nodes` list.
+For example:
++
+[source,properties]
+----
+database.port=1521
+rac.nodes=192.168.1.100,192.168.1.101
+----
+
+* Specify a value for xref:oracle-property-database-port[`database.port`], and override the default port for one or more entries in the list.
+The list can include entries that use the default `database.port` value, and entries that define their own unique port values.
+For example:
++
+[source,properties]
+----
+database.port=1521
+rac.nodes=192.168.1.100,192.168.1.101:1522
+----
+
+If you supply a raw JDBC URL for the database by using the xref:oracle-property-database-url[`database.url`] property, instead of defining a value for `database.port`, each RAC node entry must explicitly specify a port value.
```

Comment:
I don't get this sentence. Aren't `database.url` and `rac.nodes` sort of mutually exclusive? I.e. only one of them should be given?

Reply:
Yeah, that could be more clear. I think that the intent is that if you provide a value for` database.url`, and not for `database.port`, you must specify the port value for each entry in listed for the `rac.nodes` property. 

Is that right, @Naros?

Reply:
Yes, my point is: should we not reject `database.url` and `rac.nodes` being given at the same time to begin with?

Reply:
Correct, `database.url` and `rac.nodes` are not mutually exclusive.  When using the URL, we wanted to allow `database.port` to be optional allowing multiple ways to define the RAC node connection ports, either via the `database.port` setting or by including it as part of each of the RAC node entries in `rac.nodes` (or both) if one node uses a different port.

Should this be reworded?

Reply:
Ok, but when would I want to use `database.url` _and_ `rac.nodes` at the same time then?

Reply:
When your Oracle environment is deployed using Oracle RAC.  You would provide each additional RAC node `ip/host:port` in `rac.nodes` to faciliate LGWR buffer flushes on each node where-as `database.url` would point to the primary node in such a configuration.

Reply:
Ok, I see now. I thought that "all RAC nodes are equal" :)